docs/manual_installation.md

```diff
@@ -0,0 +1,168 @@
+
+To avoid possible configuration mismatches, Powergate always connects to the same network Lotus is connected. This is done automatically by Powergate which asks the Lotus node for the network identity its connected to. No extra flags or configuration are needed to select the right network to connect to. 
+
+## Indices gateway
+
+Powergate provides an indices gateway which presents miners, storage asks, and faults indices in a pretty web page.x  The listening address of this webserver can be modified with `POWD_GATEWAYHOSTADDR`/`--gatewayhostaddr`.
+
+## FFS configuration
+
+The FFS module is a central part of Powergate for storing data in Filecoin in a declarative way.
+
+To provide some context about the related configuration parameters, we should understand some basics about FFS. In the FFS module, we have two central actors: the _FFS manager_, and _FFS instances_. The _FFS Manager_ is responsible for creating and managing _FFS instances_, which provide a scoped container for storing data in Filecoin.
+
+In the usual configuration of FFS, when the manager creates a new instance it also creates a new wallet address in Lotus to be owned by it. 
+
+As an optional feature, it can fund the newly created address with initial funds. For this to happen, the manager should have a _masteraddr_. A _masteraddr_ is a wallet address owned by the manager, which is used as the source to fund newly created instances wallet addresses. 
+
+For this feature to be enabled, the `POWD_LOTUSMASTERADDR`/`--lotusmasteraddr` should be set. If this is the case, this most probably involved a previous action from the system administrator to create or import the wallet address with funds in the Lotus node. If you're running Powergate in a test network, you can avoid setting a specific _master address_, and enable the `POWD_AUTOCREATEMASTERADDR`/`--autocreatemasteraddr`. This configuration will auto-create a _master address_ for the manager, and auto-fund it with the corresponding test network facuet.
+
+Finally, to configure the amount of _attoFil_ to be transferred by the _manager_ to a newly created wallet, you should set `POWD_WALLETINITIALFUND`/`--walletinitialfund`. This configuration only applies if you set the _master address_ manually, or enabled the master address auto-creation.
+
+### Sharing wallet address in all FFS instances
+
+Depending on your use-case, having each FFS instance have its wallet address might be a beneficial setup. If you plan different FFS instances for different purposes, it may be logical to bound the amount of FIL spent on storage or retrieval actions for various purposes. Also, any abuse of FIL spending gets a hard limit per-instance and not in a _single bag_ of FIL.
```

Comment:
each FFS instance have its wallet address > each FFS instance have its _own_ wallet address

Reply:
I thought _own_ was redundant, but trust you more about English :)